rocky/Makefile

```diff
@@ -9,7 +9,7 @@ export COMPOSE_DOCKER_CLI_BUILD=1
 build: build-rocky build-rocky-frontend
 
 build-rocky:
-	docker-compose run --rm rocky make build-rocky-native
+	docker-compose run -u root --rm rocky make build-rocky-native  # root for write permissions compilemessages
```

Comment:
Compilemessages is already run in the Dockerfile. Is there any reason to run it again here or can we just remove it from the build-rocky-native step?

Reply:
The issue was the mount of the source code that removes the compiled messages when they are not present on the host (when debugging an install). Now that I am looking into the container file permissions, I realize that this might actually had to do with the host files being mounted may have had stricter permissions. I will have a look.